iOS_SDK/OneSignalSDK/Source/OneSignalWebView.m

```diff
@@ -105,8 +105,9 @@ - (void)showInApp {
     if (!navController) {
         navController = [[UINavigationController alloc] initWithRootViewController:self];
         navController.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
-        navController.presentationController.delegate = self;
     }
+    navController.presentationController.delegate = self;
```

Comment:
can you add a comment to explain this? for the future 😬

Reply:
This looks like a standard delegate setup, I am ok without having a comment in code
https://developer.apple.com/documentation/uikit/uipresentationcontroller/1618329-delegate